client/my-sites/stats/summary/index.jsx

```diff
@@ -379,11 +383,11 @@ class StatsSummary extends Component {
 							rightSection={
 								<div className="stats-module__header-nav-button">
 									<DownloadCsv
-										skipQuery={ statType === 'utm' }
 										statType={ statType }
 										query={ moduleQuery }
-										path={ path }
+										path={ statType === 'statsCountryViews' ? `locations-${ geoModeLabel }` : path }
 										period={ this.props.period }
+										skipQuery={ statType === 'statsUTM' }
```

Comment:
👍

Reply:
I think the query can be disabled everywhere and individual modules are making their own queries, I will check and remove this as part of the next PR for fixing video CSV download